package.json

```diff
@@ -36,10 +39,12 @@
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
     "mini-css-extract-plugin": "^0.5.0",
-    "css-loader": "^2.1.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-redux": "^6.0.1",
     "react-router-dom": "^5.0.0",
+    "redux": "^4.0.1",
+    "redux-promise": "^0.6.0",
```

Comment:
I noticed you made use of redux-promise instead of redux-thunk is there a reason for that.